### PR TITLE
Added Parts 5.5 & 6

### PR DESCRIPTION
**Pull request by jebikoh**
**Part 5.5:** 
 - How to make a kudu-backed impala table

**Part 6:** 
 - How to alter a kudu table (transfers over to impala table if they're backed)

Typos possible. Not tested

### CODE REVIEW

File: spark-kudu examples/scala-kudu/README.md [hunk 1]
```diff
@@ -70,15 +68,15 @@ var kuduTableName = "insert-name-here"
...
+Check if the table already exsists, and if so delete it
 ```scala
 if (kuduContext.tableExists(kuduTableName)) {
      kuduContext.deleteTable(kuduTableName)
 }
 ```
 
 ### **Step 5**: Create the schema
-Enter as many fields as needed. The format is `name, type, nullable?`. Types are SQL types and need to be imported individually. You can find a list of DataTypes [here](https://kudu.apache.org/releases/0.6.0/docs/schema_design.html). You can import these by running `import org.apache.spark.sql.types.TYPE`
+Enter as many fields as needed. Format goes `name, type, nullable?`. Types are SQL types and need to be imported induvidually. You can find a list of DataTypes [here](https://spark.apache.org/docs/2.0.0/api/java/org/apache/spark/sql/types/DataTypes.html). You can import these by running `import org.apache.spark.sql.types.TYPE`
```
Comment (nw0428) [inline spark-kudu examples/scala-kudu/README.md:79]: individually

File: spark-kudu examples/scala-kudu/README.md [hunk 2]
```diff
@@ -70,15 +68,15 @@ var kuduTableName = "insert-name-here"
 ```
 
 ### **Optional Step 4.5**: Check for exisiting table
-Check if the table already exists, and if so delete it
+Check if the table already exsists, and if so delete it
```
Comment (nw0428) [inline spark-kudu examples/scala-kudu/README.md:71]: exists

File: spark-kudu examples/scala-kudu/README.md (hunk 1 above)
Comment (nw0428) [reply spark-kudu examples/scala-kudu/README.md:79]: I preffered "The format is"

**Review by nw0428 — APPROVED** [review]
Overall LGTM

**Comment by nw0428** [comment]
Merged in https://github.com/cazena/public/pull/7

File: spark-kudu examples/scala-kudu/README.md (hunk 1 above)
Comment (jebikoh) [reply spark-kudu examples/scala-kudu/README.md:79]: Fixed

File: spark-kudu examples/scala-kudu/README.md (hunk 2 above)
Comment (jebikoh) [reply spark-kudu examples/scala-kudu/README.md:71]: Fixed

**Pull request closed without merging**